atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -20,184 +20,114 @@ export const createCrashDataMap = isTempRecord => {
           label: "Crash Date",
           editable: false,
         },
-        crash_time: {
-          label: "Crash Time",
-          editable: false,
-        },
-        day_of_week: {
+        crash_day_of_week: {
           label: "Day of Week",
+          relationshipName: "crashes_list_view",
           editable: false,
         },
         est_comp_cost_crash_based: {
           label: "Est. Comprehensive Cost",
+          relationshipName: "crash_injury_metrics_view",
           editable: false,
           format: "dollars",
         },
-        est_econ_cost: {
-          label: "Est. Economic Cost",
-          editable: false,
-          format: "dollars",
-        },
-        speed_mgmt_points: {
-          label: "Speed Management Points",
-          editable: false,
-          format: "text",
-        },
         fhe_collsn_id: {
           label: "Manner of Collision ID",
           editable: isTempRecord,
           uiType: "select",
-          lookupOptions: "atd_txdot__collsn_lkp",
-          lookupPrefix: "collsn", // We need this field so we can reference the collsn_id & collsn_desc fields in the lookup table
+          lookupOptions: "lookups_collsn_lkp",
         },
-        city_id: {
+        rpt_city_id: {
           label: "City",
           editable: true,
           uiType: "select",
-          lookupOptions: "atd_txdot__city_lkp",
-          lookupPrefix: "city",
+          lookupOptions: "lookups_city_lkp",
         },
         light_cond_id: {
           label: "Light Condition",
           editable: false,
           uiType: "select",
-          lookupOptions: "atd_txdot__light_cond_lkp",
+          lookupOptions: "lookups_light_cond_lkp",
         },
         wthr_cond_id: {
           label: "Weather Condition",
           editable: false,
           uiType: "select",
-          lookupOptions: "atd_txdot__wthr_cond_lkp",
+          lookupOptions: "lookups_wthr_cond_lkp",
         },
         obj_struck_id: {
           label: "Object Struck",
           editable: false,
           uiType: "select",
-          lookupOptions: "atd_txdot__obj_struck_lkp",
+          lookupOptions: "lookups_obj_struck_lkp",
         },
         crash_speed_limit: {
           label: "Speed Limit",
           editable: true,
           // TODO: Validate for integers
```

Comment:
i noticed that submitting anything besides an integer (including null/empty) in this field results in an uncaught Hasura error. i'll make a backlog issue to handle that.